### DISCUSSION TITLE
Auto re-enroll is disabled, how do I re-enroll after unenrolling?

### DISCUSSION THREAD

**Discussion by nullkid10** [3kh0/ext-remover#1254] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
My Chromebook is managed by my district, but it has auto re-enroll turned off and DeviceBlockDevmode = false. The problem comes when they use kiosk apps for standardized tests. How do I temporarily re-enroll into my district for tests if auto re-enroll is turned off? I’m on kernel version 4, board octopus, code name phaser. I don’t care about losing my data. Thank you!

**Comment by ghost** [comment, marked as answer]
Probably fakemurk

**Reply by nullkid10** [reply]
fakemurk doesn't work on v116+

**Reply by kosuke14** [reply]
murkmod

**Reply by ghost** [reply]
> fakemurk doesn't work on v116+

Downgrade or you can also use murkmod

**Comment by CGRmags** [comment]
get a loaner and pretend u left your Chromebook at home 💀

**Comment by kosuke14** [comment]
use sh1mmer to re-enroll maybe
after your test, unenroll
